rohmu/notifier/http.py

```diff
@@ -0,0 +1,132 @@
+"""Copyright (c) 2022 Aiven, Helsinki, Finland. https://aiven.io/"""
+from .interface import Notifier
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum, unique
+from logging import getLogger
+from queue import SimpleQueue
+from requests.exceptions import RequestException
+from typing import Any, Dict, Optional
+
+import requests
+import threading
+
+LOG = getLogger(__name__)
+
+
+@unique
+class Operation(Enum):
+    UPLOAD = "UPLOAD"
+    DELETE = "DELETE"
+    DELETE_TREE = "DELETE_TREE"
+
+
+def _get_requests_session() -> requests.Session:
+    retry = requests.adapters.Retry(
+        total=3,
+        backoff_factor=0.5,
+        status_forcelist=[429, 500, 502, 503, 504],
+        method_whitelist={"POST"},
+    )
+    adapter = requests.adapters.HTTPAdapter(max_retries=retry)
+    request_session = requests.Session()
+    request_session.mount("http://", adapter)
+    return request_session
+
+
+class HTTPNotifier(Notifier):
```

Comment:
I don't think this implementation should exist at all, there is already the `BackgroundHTTPNotifier` and this one creates unwanted behavior since it blocks on the notification query.

Reply:
if both classes are exposed and their behavior is documented, why would you claim the user configuring this is unwanted behavior?

anyways, will remove since we don't need it, and it is easy enough to add if we ever need it